GUIDE.md

```diff
@@ -0,0 +1,181 @@
+# Node Documentation
+
+Welcome to the Node.js documentation guide – or more colloquially, the docs on
+docs. If you're here, you're likely looking for guidance on how to write
```

Comment:
> That looks like an en dash. It should be an em dash. (You can use HTML entities in markdown, so I'd recommend using the mdash entity here.)

Willfix — it's easy enough to add a literal emdash here (— is Alt+Shift+hyphen on OSX.)

> No spaces before or after the dash.

I'm following the NYT usage, where dashes are surrounded in spaces. Maybe I should note that in the mechanical section?

> "Node.js Documentation Guide" rather than "Node.js documentation guide"

Willfix.

> If you think this will be read by many non-native speakers, you may be doing them a favor by avoiding words like colloquially.

Oof, that is a good point. I had imagined that this doc would be translated by i18n teams, so the translation would only have to happen once. I wonder what the best way to survey non-native speakers is?